apple/RCTMarkdownStyle.h

```diff
@@ -1,34 +1,34 @@
-#import <UIKit/UIKit.h>
+#import <React/RCTUIKit.h>
```

Comment:
I’ve put uses of RCTUIKit inside an ifdef for other libraries, because I don’t think iOS would compile since React Native Core doesn’t have it

Reply:
I'll add `#ifdef`s later on, now I'm just trying to make it compile on macOS.